sklearn/utils/optimize.py

```diff
@@ -80,29 +139,36 @@ def newton_cg(func_grad_hess, func, grad, x0, args=(), eps=1e-4, tol=1e-4,
         ``max{|g_i | i = 1, ..., n} <= tol``
         where ``g_i`` is the i-th component of the gradient.
 
-    eps : float, optional
-        If fhess is approximated, use this value for the step size.
-
     maxiter : int
-        Number of iterations.
+        Number of Newton iterations.
+
+    maxinner : int
+        Number of CG iterations.
```

Comment:
If your inner loops are less accurate, they will be faster but you will need more outer loops. It's a trade-off. If you want to be compatible with the previous code, you can set maxinner to a much larger value. This way maxinner will be mostly a safeguard. However, power users could possibly want to tweak the value: do we want to expose it?